src/app/components/videochat/jh-video-chat.html

```diff
@@ -1,65 +1,137 @@
+            <span class="glyphicon glyphicon-menu-hamburger"></span>
+          </button>
         </div>
-        <div
-          ng-if="item.content === 'Participants'"
-          class="grid-item-content"
-          id="thumbnails">
-          <div
-            jh-feed="true"
-            feed="f"
-            ng-repeat="f in vm.feeds() | orderBy:'+display'"
-            toggle-highlight-fn="vm.toggleHighlightedFeed(f)"
-            highlighted="vm.isHighlighted(f)"
-            highlighted-by-user="vm.isHighlightedByUser(f)">
-          </div>
+        <!-- Logo + Footer -->
+        <div class="hidden-xs col-sm-10 text-left">
+          <jh-footer></jh-footer>
         </div>
-        <div
-          ng-if="item.content === 'Chat'"
-          class="grid-item-content">
-          <jh-chat messages="vm.logEntries()"></jh-chat>
+        <!-- Username + Room Name + Logout -->
+        <div class="col-xs-19 col-sm-14 text-right">
+          <jh-room-info></jh-room-info>
+          <jh-main-feed-info></jh-main-feed-info>
+          <jh-leave-button></jh-leave-button>
         </div>
-      </li>
-    </ul>
+      </div>
+    </header>
   </div>
+
+  <!-- Page Content -->
+  <!-- <div id="page-content-wrapper">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-lg-12"> -->
+          <!-- <a href="#menu-toggle" class="btn btn-default" id="menu-toggle">Toggle Menu</a> -->
+        <!-- </div>
+      </div>
+    </div> -->
+
+    <!-- Activity features -->
+    <!-- <div class="row" id="videochat-body">
+      <div id="videochat-playroom">
+        <ul>
+          <li activity-item
+              row="item.position[0]" col="item.position[1]"
+              size-x="item.size[0]" size-y="item.size[1]"
+              ng-repeat="item in vm.activityItems">
+            <div
+              ng-if="item.content === 'Speaker'"
+              class="grid-item-content"
+              id="main-video"
+              jh-main-feed="true"
+              feed="vm.highlightedFeed()"
+              ng-class="{mirrored: vm.mirrored()}">
+            </div>
+            <div
+              ng-if="item.content === 'Participants'"
+              class="grid-item-content"
+              id="thumbnails">
+              <div
+                jh-feed="true"
+                feed="f"
+                ng-repeat="f in vm.feeds() | orderBy:'+display'"
+                toggle-highlight-fn="vm.toggleHighlightedFeed(f)"
+                highlighted="vm.isHighlighted(f)"
+                highlighted-by-user="vm.isHighlightedByUser(f)">
+              </div>
+            </div>
+            <div
+              ng-if="item.content === 'Chat'"
+              class="grid-item-content">
+              <jh-chat messages="vm.logEntries()"></jh-chat>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div> -->
 </div>
+
+<script>
+$("#menu-toggle").click(function(e) {
+  $("#wrapper").toggleClass("toggled");
```

Comment:
I removed this line from the script in between lines 134 & 135.

    e.preventDefault();

I am not sure if removing this is the best thing to do, but I don't think it makes any difference.